people-ops/travel-policy.html.md

```diff
@@ -24,6 +22,9 @@ We choose comfortable seats and arrangements to allow people to work during the
 for the lost time, but working is not mandatory since not everyone is comfortable working while on
 the move.
 
+If the Team member misses a trip for reasons not attributable to neither the company nor the
```

Comment:
If a team member misses a flight due to reasons not caused by the company or the carrier, they will reach the destination on their own, when possible.